gobblin-utility/src/test/java/gobblin/util/SchedulerUtilsTest.java

```diff
@@ -90,6 +91,13 @@ public void setUp()
     jobProps4.setProperty("k5", "b5");
     // test-job-conf-dir/test2/test21.pull
     jobProps4.store(new FileWriter(new File(subDir2, "test21.pull")), "");
+
+    Properties jobPropsWithInclude = new Properties();
+    jobPropsWithInclude.setProperty("include", jobsProps3File.getPath());
+    jobPropsWithInclude.setProperty("k8", "newValue8");
+    // test-job-conf-dir/test2/test22.pull
+    jobProps3.store(new FileWriter(new File(subDir2, "test22.pull")), "");
```

Comment:
Ah, interesting. Then we really need to find an alternative for commons-configuration.